tests/plugins/webp-uploads/rest-api-tests.php

```diff
@@ -33,8 +33,8 @@ static function ( $transforms ) {
 		$attachment_id = self::factory()->attachment->create_upload_object( $file_location );
 		$metadata      = wp_get_attachment_metadata( $attachment_id );
 
-		$request       = new WP_REST_Request();
-		$request['id'] = $attachment_id;
```

Comment:
PHPStan was complaining here:

```
  37     Cannot assign offset 'id' to WP_REST_Request<array>.  
  37     WP_REST_Request<array> does not accept int|WP_Error.
```

Apparently `WP_REST_Request` is not typed sufficiently in core for PHPStan to accept setting an array offset. So opting for the `set_param()` method is an easy fix for this.